doc/time-series.rst

```diff
@@ -317,13 +312,32 @@ For data indexed by a :py:class:`~xarray.CFTimeIndex` xarray currently supports:
 
 .. ipython:: python
 
-   da.to_netcdf('example.nc')
-   xr.open_dataset('example.nc')
+   da.to_netcdf('example-no-leap.nc')
+   xr.open_dataset('example-no-leap.nc')
 
 .. note::
    
-   Currently resampling along the time dimension for data indexed by a
-   :py:class:`~xarray.CFTimeIndex` is not supported.
+   While much of the time series functionality that is possible for standard
+   dates has been implemented for dates from non-standard calendars, there are
+   still some remaining important features that have yet to be implemented,
+   for example:
+
+   - Resampling along the time dimension for data indexed by a
+     :py:class:`~xarray.CFTimeIndex`
+   - Built-in plotting of data with :py:class:`cftime.datetime` coordinate axes.
+   
+   If at any time you would like to restore the old default behavior, which was
+   to attempt to decode datetimes into ``np.datetime64[ns]`` objects whenever
+   possible (regardless of calendar type), you can set ``enable_cftimeindex`` to
```

Comment:
I would rather encourage people to convert their data to a standard calendar via normal xarray/cftime APIs. That's a cleaner solution than setting global configuration.

Can we add an example of this with existing APIs? Or maybe we should augment CFTimeIndex with a new `to_datetime64` method?

Reply:
> I would rather encourage people to convert their data to a standard calendar via normal xarray/cftime APIs. That's a cleaner solution than setting global configuration.

This is a really good point and would make things a lot easier to understand.  Currently we use `xarray.coding.times.cftime_to_nptime` internally; I can convert it to a better-documented public function (perhaps with friendlier error handling too).